site/content/docs/user/known-issues.md

```diff
@@ -217,3 +218,17 @@ This problem seems to be related to a [bug in Docker][moby#9939].
 [moby#9939]: https://github.com/moby/moby/issues/9939
 [Docker resource lims]: https://docs.docker.com/docker-for-mac/#advanced
 [snap]: https://snapcraft.io/
+
+## Failures involving mistmatched kubectl versions
+
+```bash
+kubectl edit deploy -n kube-system kubernetes-dashboard
+error: SchemaError(io.k8s.api.autoscaling.v2beta1.ExternalMetricStatus): invalid object doesn't have additional properties
+```
+You can check your kubectl version by running:
```

Comment:
`kubectl version` will print the client and server versions for the current context.

maybe change to:

```
You can check your client and server versions by running:
```

Reply:
Thank you ✨